Backend/tests/test_API/api_test_user.py

```diff
@@ -43,3 +43,7 @@ def delete_user(name: str) -> Response:
 
 def patch_history_playback(user_name: str, song_name: str) -> Response:
     return client.patch(f"/users/{user_name}/playback_history/?song_name={song_name}")
+
+
+def upgrade_to_artist(name: str, headers=dict[str, str]) -> Response:
```

Comment:
Im planning on using `user_name` instead of `name` as a standar. Maybe we can start applying it here.